.travis.yml

```diff
@@ -8,6 +8,16 @@ addons:
       - gnupg2
 matrix:
   include:
+    - name: source-test
+      script:
+        - |
+          set -e
+          ./tools/prepare-source.sh
+          if [ ! $(git diff --quiet --exit-code) ]; then
+            echo
+            echo "Please include these changes in your branch: "
+            git diff
+          fi
```

Comment:
You can do multiline scripts in Travis without `sh -c`? I did not know this works.

Reply:
Welcome to the wonderful world of yaml. The `|` means everything on that level of indentation is interpreted as a [multi-line string](https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml).